performance/TestResults.js

```diff
@@ -1,7 +1,7 @@
 require('dotenv').config();
 const  fs = require('fs');
 const path = require('path');
-const fetch = require('node-fetch');
+const fetchAPI = (...args) => import('node-fetch').then(({default: fetchData}) => fetchData(...args));
```

Comment:
Could you try this?
line 4: const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
line 20: (revert the change)
(Refenece: https://github.com/node-fetch/node-fetch/blob/HEAD/docs/v3-UPGRADE-GUIDE.md#converted-to-es-module )

Or there's any reason to rename `fetchAPI`?

Reply:
Yes, we get a lint error:
error    'fetch' is already a global variable  no-shadow